mod_reforged/hooks/skills/effects/berserker_rage_effect.nut

```diff
@@ -0,0 +1,46 @@
+::mods_hookExactClass("skills/effects/berserker_rage_effect", function (o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+        this.m.Description = "This character is in a berserking rage."
```

Comment:
The vanilla file overwrites `getDescription` to return `"TODO"`. So you'll need to overwrite that function to return `this.m.Description`.